setup.py

```diff
@@ -134,6 +134,9 @@ def make_pybind11_extension(
     rpathflag = '-Wl,-rpath,%s/lib' % sys.exec_prefix
     if "CONDA_PREFIX" in os.environ:
         include_dirs.append(os.path.join(os.environ["CONDA_PREFIX"], "include"))
+    else:
```

Comment:
I do not quite follow this statement. Do you mean that this else statement does not make the building process avoid to use any files provided by conda? I am not going to avoid to use conda. I am going to provide some places to get the header files, once they could not be found because CONDA_PREFIX is not defined.